storey/utils.py

```diff
@@ -113,6 +113,8 @@ def url_to_file_system(url, storage_options):
         load_fs_dependencies(scheme)
         if scheme == 'v3io':
             remaining_path = parsed_url.path
+        elif scheme in ["wasb", "wasbs"]:
+            remaining_path = f'{parsed_url.netloc.split("@")[0]}{parsed_url.path}' 
```

Comment:
Can you please add a test so it's clearer what kind of URL this is for? Because it's hard to tell if this is correct or not otherwise.